main.tf

```
@@ -48,7 +50,7 @@ resource "aws_db_instance" "default" {
 
   ca_cert_identifier          = var.ca_cert_identifier
   parameter_group_name        = length(var.parameter_group_name) > 0 ? var.parameter_group_name : join("", aws_db_parameter_group.default.*.name)
-  option_group_name           = length(var.option_group_name) > 0 ? var.option_group_name : join("", aws_db_option_group.default.*.name)
+  option_group_name           = local.is_postgres ? null : length(var.option_group_name) > 0 ? var.option_group_name : join("", aws_db_option_group.default.*.name)
```

Could we do this instead ?

```suggestion
  option_group_name           = !local.is_postgres ? (length(var.option_group_name) > 0 ? var.option_group_name : join("", aws_db_option_group.default.*.name)) : null
```